ivy/functional/frontends/tensorflow/math.py

```diff
@@ -234,6 +234,11 @@ def pow(x, y, name="pow"):
     return ivy.pow(x, y)
 
 
+@to_ivy_arrays_and_back
+def reciprocal(x, name="reciprocal"):
+    return ivy.divide(ivy.array(1.0, dtype=x.dtype), x, name=name)
```

Comment:
We have an `ivy.reciprocal` function no need to implement a composition.
Also leave the parameter name unused it's fine